commit 0117e06d5a00e71d98c9ae76cb46b5a16d89507c
Author: bertin <macportsuser@macports.org>
Date:   Thu May 22 23:41:26 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/patch-silent-command-exec.diff

diff --git a/src/port1.0/portutil.tcl b/src/port1.0/portutil.tcl
index 913ac79ea..9711eacca 100644
--- a/src/port1.0/portutil.tcl
+++ b/src/port1.0/portutil.tcl
@@ -464,7 +464,7 @@ proc command_exec {args} {
     array set env [array get ${varprefix}.env_array]
     # Call the command.
     set fullcmdstring "$command_prefix $cmdstring $command_suffix"
-    ui_info "Executing: $fullcmdstring"
+    ui_debug "Executing: $fullcmdstring"
     set code [catch {system {*}$notty {*}$callback {*}$nice $fullcmdstring} result]
     # Save variables in order to re-throw the same error code.
     set errcode $::errorCode
